deploy/03-rbac.yaml

```diff
@@ -40,9 +40,9 @@ rules:
   verbs:
   - "*"
 - apiGroups:
-  - "operatorstatus.openshift.io"
+  - "config.openshift.io"
   resources:
-  - clusteroperators
+  - clusterversions
```

Comment:
i don't think this was supposed to change.  the resource type is still clusteroperators.